source/net/index.rst

```diff
@@ -67,6 +67,6 @@
    :caption: Resources
 
    resources/glossary
-   resources/release-notes
+   resources/release-notes-mainnet
```

Comment:
I think we'll have to find a better way to structure this. It's not great for the readers that we have two sets of release notes. I'll take a look at it once this has been merged.